cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -1,15 +1,17 @@
 {% set metrics_memory = "200Mi" -%}
 {% set eventer_memory = "200Mi" -%}
+{% set metrics_memory_per_node = 4 -%}
```

Comment:
I can't, because I use this constant in math down below. Unless you mean something like "metrics_memory_per_node_unit"?